app/Entirety/docker-compose.yml

```diff
@@ -0,0 +1,100 @@
+version: "3.9"
```

Comment:
Not anymore, I updated it. Only the Orion and mong-db images are added. However, the variable for the Database password needs to be called "DATABASE_PASSWORD" instead of "POSTGRES_PASSWORD" (int the step_by_step tutorials) because its called "DATABASE_PASSWORD" for the database settings in settings.py. file. And for the postges image the ports need to be exposed.

Reply:
I see that issue, thanks! Nice call to the password settings.

Reply:
Actually, `postgres` and `entirety` are in the same docker network, and the port `5432` of `postgres` is by default exposed within a docker network. Have you tried using `DATABASE_SERVER=entirety-db` instead of `DATABASE_SERVER=localhost`?